boardgame-ui/src/components/games/GamesList.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <div>
+    <PanelList>
+      <PanelListElement v-for="(game, id) in reverseSortedGames" v-bind:key="id">
+        <template v-slot:title>
+          <div class="games-headers">
+            <div class="is-size-6-mobile">{{game.board_game.name}}</div>
+            <div class="games-subtitle field is-grouped is-grouped-multiline">
+              <div class="control">
+                <div class="tags has-addons">
+                  <span class="tag is-primary"><i class="fas fa-user"></i></span>
+                  <span class="tag is-light">{{game.players.length}}</span>
+                </div>
+              </div>
+
+              <div class="control" v-if="game.duration">
+                <div class="tags has-addons">
+                  <span class="tag is-primary"><i class="fas fa-stopwatch"></i></span>
+                  <span class="tag is-light"><BgcDuration :duration="game.duration" /></span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </template>
+
+        <template v-slot:img>
+          <figure class="image is-64x64 is-rounded background" :style="{backgroundImage: `url('${game.board_game.thumbnail}')`}"></figure>
+        </template>
+
+        <template v-slot:content>
+          <div>
+            <RankingTable
+            :rankingMethod="game.ranking_method"
+            :data="formattedRanking(game)"></RankingTable>
+          </div>
+        </template>
+
+        <template v-slot:buttons>
```

Comment:
Do we display these buttons in any case ? for example, does not seem right to have the possibility to edit or delete a friend's game in which the current user was not a player. I don't know if there is already a policy regarding those rights in backend?

Reply:
I think it would be best to disable them altogether when on another player's page, even if the current user does have to rights to modify this game.